### PR TITLE
This is intial commit for URLHaus connector

### PR DESCRIPTION
**Pull request by IslamkamalBaker**
#### Descriptions:
_Add PR descriptions here_

#### Fix:
_Add fix/changes here_

### CODE REVIEW

File: urlhaus/info.json [hunk 1]
```diff
@@ -0,0 +1,385 @@
+{
+    "name": "urlhaus",
+    "version": "1.0.1",
+    "label": "Url Haus",
+    "description": "URLhaus is a project operated by abuse.ch. The purpose of the project is to collect, track and share malware URLs, helping network administrators and security analysts to protect their network and customers from cyber threats.",
+    "publisher": "Community",
```
Comment (cs-utkarsh-surana) [inline urlhaus/info.json]: Please change the publisher to Fortinet SE

File: urlhaus/connector.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+"""
+Copyright start
+Copyright (C) 2008 - 2021 Fortinet Inc.
```
Comment (cs-utkarsh-surana) [inline urlhaus/connector.py]: please change the copyright year to 2008 - 2023 in each file

File: urlhaus/info.json [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+                        "urlhaus_reference": "",
+                        "filename": "",
+                        "firstseen": "",
+                        "lastseen": ""
+                    }
+                ]
+            },
+            "parameters": [
+                {
+                    "title": "Hash name",
```
Comment (cs-utkarsh-surana) [inline urlhaus/info.json]: The first letter of every word in the title needs to be capital so, please change it to Hash Name.
And do the same for every title.

File: urlhaus/info.json [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+        "fields": [
+            {
+                "title": "Server URL",
+                "type": "text",
+                "name": "server_url",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "https://urlhaus-api.abuse.ch"
+            }
```
Comment (cs-utkarsh-surana) [inline urlhaus/info.json:20]: Verify SSL parameter missing in configuration parameter.

File: urlhaus/info.json [hunk 5]
```diff
@@ -0,0 +1,385 @@
...
+            "enabled": true,
+            "open": false
+        },
+        {
+            "operation": "get_signature",
+            "title": "Query signature information ",
+            "description": "URLhaus tries to identify the malware family of a payload (malware sample) served by malware URLs. Unlink tags, the signature is something that the reporter of the malware URL can not influence. To retrieve information about a signature",
+            "category": "investigation",
+            "annotation": "get_signature",
+            "output_schema": {},
```
Comment (cs-utkarsh-surana) [inline urlhaus/info.json]: Please add output schema.

File: urlhaus/info.json [hunk 6]
```diff
@@ -0,0 +1,385 @@
...
+                        "takedown_time_seconds": "",
+                        "tags": [
+                            ""
+                        ]
+                    }
+                ]
+            },
+            "parameters": [
+                {
+                    "title": "host",
```
Comment (cs-utkarsh-surana) [inline urlhaus/info.json]: Change it to Host.

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

File: urlhaus/info.json (hunk 1 above)
Comment (IslamkamalBaker) [reply urlhaus/info.json]: Change has been done

File: urlhaus/connector.py (hunk 2 above)
Comment (IslamkamalBaker) [reply urlhaus/connector.py]: Change has been done

File: urlhaus/info.json (hunk 3 above)
Comment (IslamkamalBaker) [reply urlhaus/info.json]: Change has been done for all items

File: urlhaus/info.json (hunk 5 above)
Comment (IslamkamalBaker) [reply urlhaus/info.json]: done i add it

File: urlhaus/info.json (hunk 4 above)
Comment (IslamkamalBaker) [reply urlhaus/info.json:20]: no need due to validate internet connectivity only

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**